dialoptions.go

```diff
@@ -374,8 +374,7 @@ func init() {
 // is returned by f, gRPC checks the error's Temporary() method to decide if it
 // should try to reconnect to the network address.
 //
-// Deprecated: use WithContextDialer instead.  Will be supported throughout
-// 1.x.
+// Deprecated: use WithContextDialer instead. Will be supported throughout 1.x.
```

Comment:
Please re-wrap this to avoid the unnecessary diff.

Reply:
I don't understand why you want 2 spaces separating the sentence in the docs. But anyway done.